project/home/templates/home/home_page.html

```diff
@@ -9,5 +9,12 @@
 <p>Welcome to Wagtail Social.</p>
 
 <div class="intro">{{ page.intro }}</div>
+{% for invite_friend in invite_friends %}
+    {% with invite_friend=invite_friend.specific %}
+        <div style="text-align: right;">
+            <h4><a href="{% pageurl invite_friend %}">{{ invite_friend.title }}</a></h4>
```

Comment:
Use semantic markup for headers. I.e. use the correct header hierarchy for the document rather than using lower-level headers to change font size.

- `h1` - there should be only one `h1` per page
  - `h2` - there should be `h1` above `h2`
    - `h3` - there should be `h2` through `h1` above `h3`
      - `h4` - there should be `h1` through `h3` above `h4`

```suggestion
            <h2><a href="{% pageurl invite_friend %}">{{ invite_friend.title }}</a></h2>
```